_api_app/app/Sites/multisites.twig

```diff
@@ -1,29 +1,14 @@
-{% if multisitesOption|length  > 1 %}
-  <ul id="multisites">
-    {% for multisitesOption in siteName => site %}
-      {% if environment == 'engine' || multisiteOption != siteName || ( siteName =='0' && multisiteOption !='' ) %}
-        <li {% if multisiteOption === siteName || ( siteName =='0' && multisiteOption =='' ) %} class="selected" {% endif %}>
-          <a href="{{ bertaLink site = siteName }}">
-            {% if site['title']['value'] !='' %}
-              {{ site['title']['value'] }}
-            {% else %}
-              {% if siteName =='0' %}
-                Main Site
-              {% else %}
-                {{ siteName }}
-              {% endif %}
-            {% endif %}
+<ul id="multisites">
+  {% for sites in multisite %}
+    {% if sites.isPublished is same as (true) %}
+      {% if sites.isAvailable is same as (true) %}
+        <li {% if sites.isCurrentSite is same as (true)  %} class="selected" {% endif %}>
```

Comment:
Do not white any logic in template if possible.
Add the required class for item in the `getViewData` method. You will also want to filter sites there.